django/db/migrations/graph.py

```diff
@@ -178,16 +176,12 @@ def remove_replaced_nodes(self, replacement, replaced):
         replaced = set(replaced)
         try:
             replacement_node = self.node_map[replacement]
-        except KeyError as exc:
-            exc_value = NodeNotFoundError(
+        except KeyError as err:
```

Comment:
I wonder if you made any conscious choice between `exc` and `err`. Just seems odd to change this one when other places use `exc`.